<launch>
  <!-- DEFINITION OF ARGUMENTS -->

  <!-- lidar_rotation_motor -->
  <arg name="angleIncrement" default="0.5"/>
  <arg name="motorFakeMode" default="false"/>
  <arg name="pwmFrequency" default="150"/>
  <arg name="angleRange" default="180.0"/>

  <!-- lidar_rotation_motor -->
  <node pkg="lidar_pointcloud_scan" exec="lidar_rotation_motor" name="lidar_rotation_motor" output="screen">
    <param name="angleIncrement" value="$(var angleIncrement)"/>
    <param name="motorFakeMode" value="$(var motorFakeMode)"/>
    <param name="pwmFrequency" value="$(var pwmFrequency)"/>
    <param name="angleRange" value="$(var angleRange)"/>
  </node>

</launch>